app/models/miq_request_workflow.rb

```diff
@@ -33,6 +33,16 @@ def self.all_encrypted_options_fields
     descendants.flat_map(&:encrypted_options_fields).uniq
   end
 
+  def self.update_requester_from_parameters(data, user)
+    return user if data[:user_name].blank?
+    new_user = User.lookup_by_identity(data[:user_name])
+
+    raise ActiveRecord::RecordNotFound unless new_user
```

Comment:
This was put here to handle a nil - but also to allow this test to pass: https://github.com/ManageIQ/manageiq/blob/master/spec/models/automation_request_spec.rb?#L41-L48